cd/auto-generate/auto-generate-controllers.sh

```diff
@@ -228,7 +229,9 @@ for CONTROLLER_NAME in $CONTROLLER_NAMES; do
 
     # Add all the files & create a GitHub commit
     git add .
-    COMMIT_MSG="Update ACK runtime to \`$ACK_RUNTIME_VERSION\`"
+    COMMIT_MSG="Update ACK runtime to \`$ACK_RUNTIME_VERSION\`
```

Comment:
How about instead of adding it here in commit message, you add it in PR template?
* https://github.com/aws-controllers-k8s/test-infra/blob/main/cd/auto-generate/gh_pr_body_new_release_template.txt
* https://github.com/aws-controllers-k8s/test-infra/blob/main/cd/auto-generate/gh_pr_body_template.txt

PR title is formed using commit message by default and I am not sure how line break will impact it.

Reply:
I'm not opposed to exporting `$ACK_RUNTIME_VERSION` and using it in template files. If this is preferred, let me know if you want the msg formatted a certain way.

Reply:
Thanks! I think adding two new lines in those template files will be great!

```
* ACK code-generator \`$ACK_RUNTIME_VERSION\` [release notes](https://github.com/aws-controllers-k8s/code-generator/releases/tags/$ACK_RUNTIME_VERSION)
* ACK runtime \`$ACK_RUNTIME_VERSION\` [release notes](https://github.com/aws-controllers-k8s/runtime/releases/tags/$ACK_RUNTIME_VERSION)
```

Reply:
++ to adding it into the PR template